clients/cpp-console/README.md

```diff
@@ -32,3 +32,50 @@ Check out how to [compile this C++ console sample for Linux on a Windows machine
 ## Building for Windows
 
 Check out the [Windows.md](docs/Windows.md) for detailed instructions.
+
```

Comment:
Some of the below should go into the Windows.md file, not the general README.md. Try to seperate Windows specific instructions from general instructions, which should remain in this README.md.

Reply:
Updated.